src/index.js

```diff
@@ -1,6 +1,31 @@
-import React from "react";
+import React, { useState } from "react";
 import ReactDOM from "react-dom";
-import "./index.css";
-import App from "./App";
+import { BrowserRouter, withRouter } from "react-router-dom";
+import Routing from "./Routing";
+import Navbar from "./Navbar";
+import "./App.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+
+function App(props) {
+
+  const [authenticated, setAuthenticated] = useState(true);
```

Comment:
here is where you set if the user is authenticated or not. The routes will change as a result.